guides/_subsections/view-archived-recordings.md

```diff
@@ -0,0 +1,75 @@
+## [Viewing Archived Recordings](#view-archived-recordings)
+There are several ways to view <i>Archived Recordings</i>. The first method is to navigate to the Archived Recordings tabs as mentioned in the previous section. See <a href="#archive-a-recording">Archive a Recording</a>.
```

Comment:
I'd drop the "in the previous section" part, Linking to that section is enough. Having indirect references between sections makes it more tedious to rearrange or drop content later, but direct links work fine because you can just search the text for the section title anchor link.